### ISSUE TITLE
httpie not working anymore on Mac OS X 10.10.2

### ISSUE THREAD

**Issue by janten** [jkbrzt/httpie#316]
```bash
Jan-Gerds-MacBook:radio janten$ http
Traceback (most recent call last):
  File "/usr/local/Cellar/httpie/0.9.2/libexec/bin/http", line 9, in <module>
    load_entry_point('httpie==0.9.2', 'console_scripts', 'http')()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 357, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 2394, in load_entry_point
    return ep.load()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 2108, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
  File "/usr/local/Cellar/httpie/0.9.2/libexec/lib/python2.7/site-packages/httpie/__main__.py", line 6, in <module>
    from .core import main
  File "/usr/local/Cellar/httpie/0.9.2/libexec/lib/python2.7/site-packages/httpie/core.py", line 22, in <module>
    from httpie.client import get_response
  File "/usr/local/Cellar/httpie/0.9.2/libexec/lib/python2.7/site-packages/httpie/client.py", line 15, in <module>
    urllib3.disable_warnings()
AttributeError: 'module' object has no attribute 'disable_warnings'
```

**Comment by heydonovan** [comment]
It's from my understanding that the issue comes from the ```requests``` library. What version do you have installed? You can find out with ```pip freeze | grep requests```. It might be better to run the ```disable_warnings``` method in a try/except.

**Comment by sigmavirus24** [comment]
This looks to have been installed with brew. I'm not certain how likely it is that this will be easy to debug.

**Comment by jakubroztocil** [comment]
Interesting. The formula [specifies Requests 2.5.2](https://github.com/Homebrew/homebrew/blob/68e81fece1697153f7ae6514b4895b6fd80bff19/Library/Formula/httpie.rb#L23) as a dependency which includes `urllib3` with that [function](https://github.com/jakubroztocil/httpie/blob/1a43c0e5f74fdae39a34aa76f94aa7be4015a367/httpie/client.py#L14-15). 

I guess [the import](https://github.com/jakubroztocil/httpie/blob/1a43c0e5f74fdae39a34aa76f94aa7be4015a367/httpie/client.py#L6) picks up another, older version of Requests installed elsewhere.

**Comment by jakubroztocil** [comment]
@janten what output do these two commands give you?

```bash
$ grep __version__  /usr/local/Cellar/httpie/0.9.2/libexec/vendor/lib/python2.7/site-packages/requests/__init__.py
```

```bash
$ grep disable_warnings /usr/local/Cellar/httpie/0.9.2/libexec/vendor/lib/python2.7/site-packages/requests/packages/urllib3/__init__.py
```

**Comment by janten** [comment]
Hello everyone and thanks for your help. @sigmavirus24 is correct, I installed `httpie` with `brew install httpie`. I recently installed iPython, maybe this has overwritten `requests`? In any case, here are the outputs of the commands you proposed.

```
$ pip freeze | grep requests
requests==2.3.0
```

```
$ grep __version__  /usr/local/Cellar/httpie/0.9.2/libexec/vendor/lib/python2.7/site-packages/requests/__init__.py
__version__ = '2.5.2'
```

```
$ grep disable_warnings /usr/local/Cellar/httpie/0.9.2/libexec/vendor/lib/python2.7/site-packages/requests/packages/urllib3/__init__.py
def disable_warnings(category=exceptions.HTTPWarning):
```

**Comment by jakubroztocil** [comment]
@janten thanks, that looks good.  And what about

```bash
$ echo $PYTHONPATH
```
?

**Comment by janten** [comment]
```bash
$ echo $PYTHONPATH

```
(seems like $PYTHONPATH is not set)

```bash
$ which http
/usr/local/bin/http
```

```bash
$ cat "`which http`"
#!/bin/bash
PYTHONPATH="/usr/local/Cellar/httpie/0.9.2/libexec/lib/python2.7/site-packages:/usr/local/Cellar/httpie/0.9.2/libexec/vendor/lib/python2.7/site-packages" exec "/usr/local/Cellar/httpie/0.9.2/libexec/bin/http" "$@"
```

**Comment by jm3** [comment]
I'm having the same issue, with similar results:
```bash
pip freeze | grep requests
requests==2.3.0

grep __version__  /usr/local/Cellar/httpie/0.9.2_1/libexec/vendor/lib/python2.7/site-packages/requests/__init__.py
__version__ = '2.6.0'

grep disable_warnings /usr/local/Cellar/httpie/0.9.2_1/libexec/vendor/lib/python2.7/site-packages/requests/packages/urllib3/__init__.py
def disable_warnings(category=exceptions.HTTPWarning):

echo $PYTHONPATH

which http
/usr/local/bin/http

cat "`which http`"
#!/bin/bash
PYTHONPATH="/usr/local/Cellar/httpie/0.9.2_1/libexec/lib/python2.7/site-packages:/usr/local/Cellar/httpie/0.9.2_1/libexec/vendor/lib/python2.7/site-packages" exec "/usr/local/Cellar/httpie/0.9.2_1/libexec/bin/http" "$@"
```

Anything else I can do to help debug this?

**Comment by jakubroztocil** [comment]
@jm3 could you try to uninstall the globally-installed Requests and try it again? 

```bash
$ pip uninstall requests
```

**Comment by jm3** [comment]
@jakubroztocil that did the trick! thanks.

**Comment by jakubroztocil** [comment]
Great. I wonder if it would be possible to prevent this from happening by tweaking [the formula](https://github.com/Homebrew/homebrew/blob/68e81fece1697153f7ae6514b4895b6fd80bff19/Library/Formula/httpie.rb).

**Comment by anentropic** [comment]
I'm getting this error without any homebrew install (pip installed, inside a virtualenv)

According to pip, httpie requires `requests >= 2.3.0`

I have requests 2.3.0 installed and it doesn't work

doing a `pip install --upgrade httpie` brings requests 2.7.0 and now it works

does the setup.py need updating?

**Comment by maryburford** [comment]
y'all nerds

**Comment by itkach** [comment]
I am getting the same error, also installed with pip in a virtualenv:
```
% pip install httpie
Downloading/unpacking httpie
  Downloading httpie-0.9.2-py2.py3-none-any.whl (66kB): 66kB downloaded
Requirement already satisfied (use --upgrade to upgrade): requests>=2.3.0 in ./.virtualenvs/t/lib/python2.7/site-packages (from httpie)
Requirement already satisfied (use --upgrade to upgrade): Pygments>=1.5 in ./.virtualenvs/t/lib/python2.7/site-packages (from httpie)
Installing collected packages: httpie
Successfully installed httpie
```

```
% pip freeze | grep requests
requests==2.3.0
```

**Comment by jeanferri** [comment]
@anentropic may you have an older version of urllib3 without disable_warnings method and using it instead?

**Comment by hangtwenty** [comment]
to @jeanferri 's point, `pip install --upgrade urllib3` may also be necessary for some users

**Comment by sigmavirus24** [comment]
@hangtwenty if installing everything from pip, then that will not help anyone.

**Comment by hangtwenty** [comment]
@sigmavirus24 OK nevermind, I was trying to relay something said by someone at work using Windows. I don't use Windows myself so I couldn't validate. Deleted comment so it won't mislead anyone

**Comment by jkbrzt** [comment]
This issue should be fixed in 0.9.3.

**Comment by bheemreddy181** [comment]
locking httpie to 0.9.4 resolves this issue 

```
 pip install httpie==0.9.4
```
